e2e/vmcompose/provider.go

```
@@ -282,12 +296,17 @@ func (p *Provider) StartNodes(ctx context.Context, _ ...*e2e.Node) error {
 	var onceErr error
 	p.once.Do(func() {
 		log.Info(ctx, "Copying artifacts to VMs")
+		timestampDir := time.Now()
+		identifier := fmt.Sprintf("DEPLOY%s", timestampDir.Format(time.RFC3339))
 		for vmName := range p.Data.VMs {
 			err := copyToVM(ctx, vmName, p.Testnet.Dir)
 			if err != nil {
-				onceErr = errors.Wrap(err, "copy files", "vm", vmName)
+				onceErr = errors.Wrap(err, "copy files to VM", "vm", vmName)
 				return
 			}
+			go func(vmName string) {
+				_ = copyToGCP(ctx, vmName, p.Testnet.Dir, p.Testnet.Name, identifier)
```

never ignore errors, at least so you can see what is going wrong